docs/balloon.md

```diff
@@ -0,0 +1,74 @@
+# Balloon
+
+Cloud Hypervisor emulates a balloon device based on the VIRTIO specification.

```

Comment:
There is no emulation here. This is a paravirtualised device and no such physical device type exists.

Reply:
Yes definitely wrong wording. Do you want me to replace `emulates` with `implements`?